aruco_markers:
  - id: 0
    distances_to_axes:
      axis_1:
        x: 0.1
        y: 0.2
        z: 0.6
      axis_2:
        x: 0.3
        y: 0.2
        z: 0.8
  - id: 1
    distances_to_axes:
      axis_1:
        x: 0.2
        y: 0.3
        z: 0.5
      axis_2:
        x: 0.4
        y: 0.1
        z: 0.7
  - id: 2
    distances_to_axes:
      axis_1:
        x: 0.15
        y: 0.25
        z: 0.55
      axis_2:
        x: 0.35
        y: 0.15
        z: 0.65
  - id: 3
    distances_to_axes:
      axis_1:
        x: 0.1
        y: 0.2
        z: 0.6
      axis_2:
        x: 0.3
        y: 0.2
        z: 0.8
  - id: 4
    distances_to_axes:
      axis_1:
        x: 0.25
        y: 0.35
        z: 0.45
      axis_2:
        x: 0.45
        y: 0.25
        z: 0.75
  - id: 5
    distances_to_axes:
      axis_1:
        x: 0.22
        y: 0.32
        z: 0.52
      axis_2:
        x: 0.42
        y: 0.12
        z: 0.72
  - id: 6
    distances_to_axes:
      axis_1:
        x: 0.18
        y: 0.28
        z: 0.58
      axis_2:
        x: 0.38
        y: 0.18
        z: 0.68
  - id: 7
    distances_to_axes:
      axis_1:
        x: 0.12
        y: 0.22
        z: 0.62
      axis_2:
        x: 0.32
        y: 0.22
        z: 0.82
  - id: 8
    distances_to_axes:
      axis_1:
        x: 0.27
        y: 0.37
        z: 0.47
      axis_2:
        x: 0.47
        y: 0.27
        z: 0.77
  - id: 9
    distances_to_axes:
      axis_1:
        x: 0.19
        y: 0.29
        z: 0.59
      axis_2:
        x: 0.39
        y: 0.19
        z: 0.69
  - id: 10
    distances_to_axes:
      axis_1:
        x: 0.21
        y: 0.31
        z: 0.51
      axis_2:
        x: 0.41
        y: 0.11
        z: 0.71
  - id: 11
    distances_to_axes:
      axis_1:
        x: 0.23
        y: 0.33
        z: 0.53
      axis_2:
        x: 0.43
        y: 0.13
        z: 0.73
  - id: 12
    distances_to_axes:
      axis_1:
        x: 0.24
        y: 0.34
        z: 0.44
      axis_2:
        x: 0.44
        y: 0.24
        z: 0.74
  - id: 13
    distances_to_axes:
      axis_1:
        x: 0.26
        y: 0.36
        z: 0.46
      axis_2:
        x: 0.46
        y: 0.26
        z: 0.76
  - id: 14
    distances_to_axes:
      axis_1:
        x: 0.28
        y: 0.38
        z: 0.48
      axis_2:
        x: 0.48
        y: 0.28
        z: 0.78
  - id: 15
    distances_to_axes:
      axis_1:
        x: 0.29
        y: 0.39
        z: 0.49
      axis_2:
        x: 0.49
        y: 0.29
        z: 0.79
  - id: 16
    distances_to_axes:
      axis_1:
        x: 0.3
        y: 0.4
        z: 0.5
      axis_2:
        x: 0.5
        y: 0.3
        z: 0.8
  - id: 17
    distances_to_axes:
      axis_1:
        x: 0.31
        y: 0.41
        z: 0.51
      axis_2:
        x: 0.51
        y: 0.31
        z: 0.81
  - id: 18
    distances_to_axes:
      axis_1:
        x: 0.32
        y: 0.42
        z: 0.52
      axis_2:
        x: 0.52
        y: 0.32
        z: 0.82
  - id: 19
    distances_to_axes:
      axis_1:
        x: 0.33
        y: 0.43
        z: 0.53
      axis_2:
        x: 0.53
        y: 0.33
        z: 0.83

axes:
  - id: 0
    distance_to_next_axis: 0.1
  - id: 1
    distance_to_next_axis: 0.2
  - id: 2
    distance_to_next_axis: 0.3
  - id: 3
    distance_to_next_axis: 0.4

cameras:
  - id: 0
    distance_to_origin:
      x: -0.3
      y: -4.4
      z: 3.3
    euler_angles:
      x: 0
      y: 50
      z: 0
